lib/floe/container_runner/docker.rb

```diff
@@ -30,7 +30,8 @@ def run_async!(resource, env = {}, secrets = {}, _context = {})
         end
 
         begin
-          runner_context["container_ref"] = run_container(image, env, runner_context["secrets_ref"])
+          result = run_container(image, env, runner_context["secrets_ref"])
+          runner_context["container_ref"] = result.strip
```

Comment:
I'm fine with the strip, but I wonder if run_container method should just do a `chomp` at the end regardless - that bonus newline might affect a bunch of things.

Reply:
Yeah that's true I can do that instead